tests/output/sapi_windows_vt100_support_stream_lose_data.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+sapi_windows_vt100_support(): casting stream emits data loss
+--SKIPIF--
+<?php
+if(PHP_OS_FAMILY !== "Windows") {
+    echo "skip Only for Windows systems";
+} elseif (version_compare(
+    PHP_WINDOWS_VERSION_MAJOR.'.'.PHP_WINDOWS_VERSION_MINOR.'.'.PHP_WINDOWS_VERSION_BUILD,
+     '10.0.10586'
+) >= 0) {
```

Comment:
In my opinion, this formatting is terrible. Only checking the build number should be sufficient though (see https://en.wikipedia.org/wiki/List_of_Microsoft_Windows_versions):
```suggestion
} elseif (version_compare(PHP_WINDOWS_VERSION_BUILD, '10586') >= 0) {
```